tests/security/ima/ima_measurement_audit.pm

```diff
@@ -42,7 +42,13 @@ sub run {
 
         # We do not check the exact file hash here, but to ensure the audit
         # record existed
-        assert_script_run("ausearch -m INTEGRITY_RULE |grep '$f->{file}.*hash='");
+        if (is_sle('>=16')) {
+            # skipping audit check on SLE 16 due to bsc#1247246
+            record_soft_failure('SKIPPING TEST; bsc#1247246');
+            return;
+        } else {
```

Comment:
The `else` clause is redundant. It can look like this:

```
return record_soft_failure('SKIPPING TEST; bsc#1247246') if is_sle('>=16');
assert_script_run("ausearch -m INTEGRITY_RULE | grep '$f->{file}.*hash='");
```